app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala

```diff
@@ -59,11 +59,30 @@ class CompaniesHouseStubController extends InjectedController {
 
     Action {
       companyNumber match {
-        case "00000002" => Ok(stubLtdPartnershipProfileWithNonMatchingPostcode)
         case "00000001" => NotFound
+        case "00000002" => Ok(stubLtdPartnershipProfileWithNonMatchingPostcode)
+        case "00000003" => Ok(stubLtdPartnershipProfileWithNameLongerThan105Chars(companyNumber))
         case _ => Ok(stubLtdPartnershipProfile)
       }
     }
   }
+
+  //noinspection ScalaStyle
+  private def stubLtdPartnershipProfileWithNameLongerThan105Chars(companyNumber: String): JsObject = {
```

Comment:
We know the company number is going to be 00000003 because of the matching above so this could be a val instead inside the method?